src/entity/Game.java

```diff
@@ -0,0 +1,31 @@
+package entity;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public interface Game {
+
+    // GETTERS
+    String getID();
+    String getDifficulty();
+    String getGenre();
+    int getInitialLives();
+    int getMaxRounds();
+    int getCurrentLives();
+    int getRoundsPlayed();
+    int getScore();
+    Round getRound();
```

Comment:
As per our design docs, this should be `Round getCurrentRound();` since what matters to us is having a way to get the current round.